apiserver/pkg/server/job_server.go

```diff
@@ -0,0 +1,99 @@
+package server
+
+import (
+	"context"
+	"github.com/ray-project/kuberay/apiserver/pkg/manager"
+	"github.com/ray-project/kuberay/apiserver/pkg/model"
+	"github.com/ray-project/kuberay/apiserver/pkg/util"
+	api "github.com/ray-project/kuberay/proto/go_client"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+	"google.golang.org/protobuf/types/known/emptypb"
+)
+
+// implements `type RayJobServiceServer interface` in job_grpc.pb.go
+// RayJobServer is the server API for RayJobServer service.
+
+func NewRayJobServer(resourceManager *manager.ResourceManager) *RayJobServer {
+	return &RayJobServer{resourceManager: resourceManager}
+}
+
+
+// TODO(basasuya) add event outputs
+type RayJobServer struct {
+	resourceManager *manager.ResourceManager
+	api.UnimplementedRayJobServiceServer
+}
+
+
+func (s *RayJobServer) CreateRayJob(ctx context.Context, request *api.CreateRayJobRequest) (*api.RayJob, error) {
+	// use the namespace in the request to override the namespace in the job definition
+	request.Job.Namespace = request.Namespace
```

Comment:
for creation, let's do a validation first. check how raycluster did and we can raise some 400 issues.

Reply:
@Basasuya This is necessary to reduce error creation in cluster. Let's add some validation here.

Reply:
after manual test, job create/list/delete would be OK. but after ray cluster create, seems job can not run through.

Reply:
I feel that's a job configuration issue. It works on my side and let's discuss this issue offline.